xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/EditForm.java

```diff
@@ -474,6 +500,26 @@ public List<String> getTemporaryUploadedFiles()
         return temporaryUploadedFiles;
     }
 
+    /**
+     * @return the value of the {@code activateRequiredRights} request parameter
+     * @since 15.4RC1
+     */
+    @Unstable
+    public String getActivateRequiredRights()
```

Comment:
Would make more sense for this to return a boolean value and to check there that it's equals to 1 no?